model_bakery/random_gen.py

```diff
@@ -133,10 +133,6 @@ def gen_slug(max_length: int) -> str:
 gen_slug.required = ["max_length"]  # type: ignore[attr-defined]
 
 
-def gen_text() -> str:
-    return gen_string(MAX_LENGTH)
-
```

Comment:
Deleted as no longer in use

Reply:
I would first deprecate this generator, as this is a public interface and used in other codebases:
https://github.com/search?q=random_gen.gen_text+language%3APython&type=code

Would you mind, instead of deleting it, adding a deprecation warning here?
https://docs.python.org/3/library/exceptions.html#DeprecationWarning

If you are not sure how to tackle it, I'd suggest just skipping the deletion - I will take care of the deprecation.

Reply:
Thank you so much @amureki I didn't think about it. I added deprecation warning. Please, check if everything looks good to you!